Tasks/CopyFiles/CopyFiles.js

```diff
@@ -110,6 +110,10 @@ for (var i = 0; i < allPaths.length; i++) {
         allFiles.push(allPaths[i]);
     }
 }
+// if we only have exclude filters, we need add a include all filter, so we can have something to exclude.
+if (includeContents && includeContents.length == 0 && excludeContents && excludeContents.length > 0) {
```

Comment:
We create includeContents and excludeContents above so I think we can simplify this condition.